metadata-ingestion/src/datahub/ingestion/source/kafka/kafka_connect.py

```diff
@@ -1178,9 +1170,16 @@ def get_connectors_manifest(self) -> List[ConnectorManifest]:
 
         payload = connector_response.json()
 
-        for c in payload:
-            connector_url = f"{self.config.connect_uri}/connectors/{c}"
-            connector_response = self.session.get(connector_url)
+        for connector_name in payload:
+            try:
+                connector_url = f"{self.config.connect_uri}/connectors/{connector_name}"
+                connector_response = self.session.get(connector_url)
+                connector_response.raise_for_status()
+            except Exception as e:
+                self.report.warning(
+                    "Failed to get connector details", connector_name, exc=e
+                )
+                continue
```

Comment:
similar to `_get_connector_(topics|tasks)`, 
what about wrapping this in a `_get_connector_manifest` method too?